spec/indexers/sufia/file_set_indexer_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'spec_helper'
+
+describe Sufia::FileSetIndexer do
+  let(:user) { create(:user) }
+  let!(:file_set) { create(:file_set, user: user) }
+  let(:service) { described_class.new(file_set) }
+  let(:file) { File.open(fixture_path + '/world.png') }
+
+  before do
+    Hydra::Works::AddFileToFileSet.call(file_set, file, :original_file)
+    file_set.original_file.page_count = ['1']
+    file_set.original_file.file_title = ['title']
+    file_set.original_file.duration = ['0:1']
+    file_set.original_file.sample_rate = ['sample rate']
+  end
+  subject { service.generate_solr_document }
+
+  it 'indexes FileSet ids separate from other memebers and resource type' do
```

Comment:
Not sure the spec description matches the assertions in the test, looks like copypasta from the work indexer spec?